modules/perform.cpp

```diff
@@ -90,13 +90,15 @@ class CPerform : public CModule {
 	MODCONSTRUCTOR(CPerform) {
 		AddHelpCommand();
 		AddCommand("Add",     static_cast<CModCommand::ModCmdFunc>(&CPerform::Add),
-			"<command>");
+			"<command>", "Add a command to be sent to the server on connect.");
 		AddCommand("Del",     static_cast<CModCommand::ModCmdFunc>(&CPerform::Del),
-			"<nr>");
-		AddCommand("List",    static_cast<CModCommand::ModCmdFunc>(&CPerform::List));
-		AddCommand("Execute", static_cast<CModCommand::ModCmdFunc>(&CPerform::Execute));
+			"<number>", "Delete command <number> in the output of 'List'.");
+		AddCommand("List",    static_cast<CModCommand::ModCmdFunc>(&CPerform::List),
+			"", "List the currently configured commands.");
+		AddCommand("Execute", static_cast<CModCommand::ModCmdFunc>(&CPerform::Execute),
+			"", "Send the commands to the server now.");
 		AddCommand("Swap",    static_cast<CModCommand::ModCmdFunc>(&CPerform::Swap),
-			"<nr> <nr>");
+			"<cmd1> <cmd2>", "Modify the order in which the commands are sent.");
```

Comment:
The problem is not the abbreviations, but changing between command and cmd everywhere.

Windows command interpreter has been cmd for years, but I don't believe it has ever been command. In this pull request you use word "command" and abbreviation "cmd" and I am saying that to be consistent you must either decide that you use "command" or "cmd".

And I am suggesting "command" because it's more clear to not-so-advanced users who e.g. use ZNC provider X.